load.php

```diff
@@ -11,16 +11,17 @@
 require_once __DIR__ . '/inc/namespace.php';
 
 add_action( 'hm-platform.modules.init', function () {
+	$is_cloud = in_array( get_environment_architecture(), [ 'ec2', 'ecs' ], true );
 	$default_settings = [
-		'enabled'            => in_array( get_environment_architecture(), [ 'ec2', 'ecs', 'local-server' ] ),
+		'enabled'            => true,
 		'cavalcade'          => true,
 		's3-uploads'         => true,
-		'aws-ses-wp-mail'    => true,
-		'batcache'           => true,
+		'aws-ses-wp-mail'    => $is_cloud,
+		'batcache'           => $is_cloud,
```

Comment:
I'm wondering if it would be better to put this on anyway? Having page caching locally is annoying when you don't want it, but we probably do want developers to be maybe be getting it by default?

Reply:
I'd say no, you can always config it on if needed.

Reply:
I'm generally ok with it being on as you rarely generate enough traffic to trigger batcache locally unless it's intentional. I prefer it on personally as it can show potential issues with batcache locally however I'm fine if it's something we just need to turn on manually as it means you've done so mindfully and intentionally.

Reply:
I'd say better to be explicit about turning it on, as debugging it otherwise can be a total pain